### ISSUE TITLE
need simplify when condition for redhat OS family

### ISSUE THREAD

**Issue by roumano** [dhoeric/ansible-aws-ssm#5]
need simplify when condition for redhat OS family
this test condition : 
```
(ansible_distribution == 'Amazon' or ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux')
```
can be much more simple with ansible_os_family variable : 
```
ansible_os_family == 'RedHat'
```

**Comment by dhoeric** [comment]
will handle in this week

**Comment by roumano** [comment]
fixed via #6
